docs/en/concepts-and-designs/mal.md

```diff
@@ -164,7 +164,7 @@ These operations can be used to aggregate overall label dimensions or preserve d
 Example expression:
 
 ```
-instance_trace_count.sum(by: ['az'])
+instance_trace_count.sum(['az'])
```

Comment:
@hanahmily @wankai123 Do you remember when we had by as keyword? I remember it doesn't there from beginning.

Reply:
This is the language spec of Groovy. The parameter name of sum function is by, which can be omitted or can be added explicitly

Reply:
When I add by keyword , OAP  throw exception 
```
  - name: sv_http_connections
    exp: "apisix_nginx_http_current_connections.sum(by: ['state','host_name']).service(['host_name'] , Layer.APISIX)"

```

```
Possible solutions: sum(java.util.List), dump(), use([Ljava.lang.Object;), any(), use(java.util.List, groovy.lang.Closure), use(java.lang.Class, groovy.lang.Closure)
org.apache.skywalking.oap.meter.analyzer.dsl.ExpressionParsingException: failed to parse expression: (apisix_nginx_http_current_connections.tag({tags -> tags.host_name = 'apisix::' + tags.service_instance_id})).sum(by: ['state','host_name']).service(['host_name'] , Layer.APISIX), error:No signature of method: org.apache.skywalking.oap.meter.analyzer.dsl.SampleFamily.sum() is applicable for argument types: (LinkedHashMap) values: [[by:[state, host_name]]]
Possible solutions: sum(java.util.List), dump(), use([Ljava.lang.Object;), any(), use(java.util.List, groovy.lang.Closure), use(java.lang.Class, groovy.lang.Closure)
	at org.apache.skywalking.oap.meter.analyzer.dsl.Expression.parse(Expression.java:59) ~[classes/:?]
	at org.apache.skywalking.oap.meter.analyzer.Analyzer.build(Analyzer.java:91) ~[classes/:?]
	at org.apache.skywalking.oap.meter.analyzer.MetricConvert.lambda$new$2(MetricConvert.java:71) ~[classes/:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193) ~[?:1.8.0_321]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1384) ~[?:1.8.0_321]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482) ~[?:1.8.0_321]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472) ~[?:1.8.0_321]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708) ~[?:1.8.0_321]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_321]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499) ~[?:1.8.0_321]
	at org.apache.skywalking.oap.meter.analyzer.MetricConvert.<init>(MetricConvert.java:78) ~[classes/:?]
	at org.apache.skywalking.oap.meter.analyzer.prometheus.PrometheusMetricConverter.<init>(PrometheusMetricConverter.java:73) ~[classes/:?]
	at org.apache.skywalking.oap.server.receiver.otel.oc.OCMetricHandler.lambda$active$1(OCMetricHandler.java:188) ~[classes/:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193) ~[?:1.8.0_321]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1384) ~[?:1.8.0_321]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482) ~[?:1.8.0_321]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472) ~[?:1.8.0_321]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708) ~[?:1.8.0_321]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_321]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499) ~[?:1.8.0_321]
```

Reply:
@kezhenxu94 @hanahmily Need another confirmation.

Reply:
The expression should be `apisix_nginx_http_current_connections.sum(by = ['state','host_name']).service(['host_name'] , Layer.APISIX)`,  

see: https://github.com/apache/skywalking/blob/dd6b7a1fc0f8581e25c86cf6c904848f0c574e92/oap-server/analyzer/meter-analyzer/src/test/java/org/apache/skywalking/oap/meter/analyzer/dsl/AggregationTest.java#L76

Reply:
ACK , I  would update  the example